src/components/chats/index.js

```diff
@@ -0,0 +1,71 @@
+import useTranslation from 'hooks/useTranslation';
+import { useEffect, useState } from 'react';
+import { useGetConversationQuery } from 'services/conversation/conversation';
+import profile from 'assets/profile.svg';
+import { v4 as uuid } from 'uuid';
+
+import './styles.scss';
+
+const Chats = () => {
+  const [tabSelected, setTabSelected] = useState('HOME');
+  const user = JSON.parse(localStorage.getItem('user'));
+  const { data: matchConversations } = useGetConversationQuery();
+  const [matchList, setMatchList] = useState([]);
+  const t = useTranslation();
+  const getMatches = () => {
+    const matches = matchConversations?.matches || [];
+    setMatchList(matches);
+  };
+
+  useEffect(() => {
+    getMatches();
+  }, [matchConversations]);
+
+  const handleClickOpenChat = () => {
+    setTabSelected('CHAT');
+  };
+
+  return (
+    <>
+      <h2>{t('home.title')}</h2>
+      <img alt="profile avatar" src={profile}></img>
+      <h5>{user.username}</h5>
```

Comment:
You shouldn't use `h5` unless you've usde an `h4` on this file. You can read more about it [here](https://www.w3schools.com/html/html_headings.asp)

Reply:
Perfect 👍